### PR TITLE
140540769: Fix Properties and messages in scheduler

### PR DESCRIPTION
**Pull request by kanekoh**

### CODE REVIEW

File: scheduler/src/main/resources/application.properties [hunk 1]
```diff
@@ -1,28 +1,26 @@
...
-spring.datasource.driverClassName=org.postgresql.Driver
-spring.datasource.url=jdbc:postgresql://127.0.0.1/autoscaler
-spring.datasource.username=postgres
-spring.datasource.password=postgres
-
-#quartz job
-scalingenginejob.reschedule.interval.millisecond=10000
-scalingenginejob.reschedule.maxcount=6
-scalingengine.notification.reschedule.maxcount=3
+# datasource for application and quartz
```
Comment (fraenkel) [inline scheduler/src/main/resources/application.properties]: Is this even needed anymore? There is no content.

Comment (kanekoh) [inline scheduler/src/main/resources/application.properties]: We kept application.properties for reference only. Just to know what properties are being used.
If you think it's not required, we can remove it?

Comment (fraenkel) [inline scheduler/src/main/resources/application.properties]: Git will always remember it but if we don't need it, it is more confusing to have it.

Comment (kanekoh) [inline scheduler/src/main/resources/application.properties]: OK, We will remove it.

**Pull request merged**